dags/events_to_amplitude.py

```diff
@@ -66,3 +67,19 @@ def key_file(project):
     },
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/events_to_amplitude.sh",
     dag=dag)
+
+rocket_android_events_to_amplitude = EMRSparkOperator(
+    task_id="rocket_android_events_to_amplitude",
+    job_name="Rocket Android Events to Amplitude",
+    execution_timeout=timedelta(hours=8),
+    instance_count=ROCKET_ANDROID_INSTANCES,
+    env={
+        "date": "{{ ds_nodash }}",
+        "max_requests": ROCKET_ANDROID_INSTANCES * VCPUS_PER_INSTANCE,
+        "key_file": key_file("rocket_android"),
+        "artifact": get_artifact_url(slug, branch="add-rocket-amplitude"),
```

Comment:
I see the PR is still open – did you mean to add the task pinned to the branch?

Reply:
No definitely not. Thanks for the heads up, let me get that cleaned up and merged